src/exportAuthRecord.ts

```diff
@@ -0,0 +1,82 @@
+import { IActionContext } from '@microsoft/vscode-azext-utils';
```

Comment:
Quick question: do we need copyright headers for this repo too?

Reply:
Yes, look in other files for an example.